### ISSUE TITLE
Extend and configure translation class.

### ISSUE THREAD

**Issue by phlegx** [shioyama/mobility#421]
I want to extend the translation class to work with some other plugins. Where can I find a HowTo or can someone help me to start?

I use the table backend and I need to include some things.

**Comment by shioyama** [comment]
Thanks for reaching out! If I understand correctly, you'd like to extend the translation class for your model class to work with other (non-Mobility) plugins?

Here's where it's defined (on the `master` branch):

https://github.com/shioyama/mobility/blob/41547237ce87857d7ca198b9a13d4c6201ef39a1/lib/mobility/backends/active_record/table.rb#L244-L249

If your model name is `Post`, it defines a constant `Post::Translation` which you can simply modify:

```ruby
class Post::Translation
  # .. your custom stuff
end
```

It's a bit more tricky if you want to modify the translation class for *all* models. Mobility has a class, `Mobility::ActiveRecord::ModelTranslation` which it subclasses, so if you wanted to change the translation class for all models you could modify that one:

```ruby
class Mobility::ActiveRecord::ModelTranslation
  # ... your custom stuff
end
```

Put that in an initializer and it should work.

**Comment by shioyama** [comment]
Mind you, the namespace of `Mobility::ActiveRecord::ModelTranslation` will probably change in v1.0, in part to fix #258.
